src/lib/hooks/usePublicProjects/index.ts

```diff
@@ -69,7 +63,7 @@ export const getPublicProjects = async (
       foreignTable: "devices.records",
       ascending: false,
     })
-    .limit(1, { foreignTable: "devices" })
+    .limit(10, { foreignTable: "devices" })
```

Comment:
why return 10 devices if parseDeviceRecords `return mappedDevices[0]` only one anyways.

Reply:
We cannot be sure that the first device has data and we want to show the fancy area chart on top of the ProjectPreview card. That's why we grab the 10 first devices and check which one has the most records.